docs/User Manual.rst

```diff
@@ -0,0 +1,1014 @@
+  - padmat: material name or universe index (u<id>) that fills the possible extra space between the lattice and its bounding surface. Also the keyword ``void`` is allowed
+  - map: map that includes the universe ids of the elements of the lattice. The order is: increasing x, increasing y and then increasing z
+  - origin (*optional*, default = (0.0 0.0 0.0)): (x y z) array with the origin of the universe. [cm]
+  - rotation (*optional*, default = (0.0 0.0 0.0)): (x y z) array with the rotation angles in degrees applied to the universe. [°]
+
+Example: ::
+
+      uni_lattice { id 10; type latUniverse; shape (3 2 2); pitch (1.0 1.0 1.5); padMat u<3>; map (
+      1 2 3 // x: 1-3, y: 1, z: 1
+      4 5 6 // x: 1-3, y: 2, z: 1
+      7 8 9 // x: 1-3, y: 1, z: 2
+      10 11 12 ) } // x: 1-3, y: 2, z: 2
+
+* rootUniverse: top level universe of geometry  **NB THE DOCUMENTATION IN THE FILE IS WRONG (sample input)**
+
+  - border: id of the boundary surface for the whole geometry
+  - fill: inside filling, as a material name or a universe (u<id>)
+
+Example: ::
+  
+      root { id 1000; type rootUniverse; border 10; fill u<1>; }
+
+Visualiser
+----------
+
+To **plot** a geometry, the keyword ``viz`` must be present in the input file: ::
+
+      viz {
+      <name1> { type <vizType>; *keywords* }
+      <name2> { type <vizType>; *keywords* }
+      }
+      
+The possible types of files that the geometry is plotted in are:
+
+vtk
+###
+
+* corner: (x y z) array with the corner of the geometry [cm]
+* width: (x y z) array with the width of the mesh in each direction [cm]
+* vox: (x y z) array with the number of voxels requested in each direction **NOTE: the sample input in outputVTK doesn't have vox**
+* what (*optional*, default = material): defines what is highlighted in the plot; options are ``material`` and ``cellID``
+
+Example: ::
+
+      plotVTK { type vtk; corner (10.0 6.0 2.0); width (20.0 12.0 4.0); vox (4000 120 400); what cellID; }
+
+bmp
+###
+
+* centre: (x y z) array with the coordinates of the center of the plot [cm]
+* axis: ``x``, ``y`` or ``z``, it's the axis normal to the 2D plot
+* width (*optional*, default = whole geometry): (y z), (x z) or (x y) array with the width of the geometry plotted in each direction [cm]
+* res: (y z), (x z) or (x y) array with the resolution of the mesh in each direction
+* output: name of the output file, with extension ``.bmp``
+* what (*optional*, default = material): defines what is highlighted in the plot; options are ``material`` and ``cellID``
+
+Example: ::
+
+      plotBMP { type bmp; axis z; width (50 10); res (1000 200); output geomZ; what material; }
+      
+.. note:: 
+   SCONE can be run to visualise geometry without actually doing transport, by including ``--plot`` when running the application. In this case the visualiser has to be included in the file.
+
+**WHY ARE VTK AND BMP SCATTERED AROUND IN THE CODE???**
```

Comment:
You mean why VTK lives in `Visualisation` and BMP in `UserInterface`?

Good question... I am not quite sure why I put BMP where it is...

Reply:
Yes, while I was looking for the files I had troubles finding BMP, while it would seem natural to have them in the same place. These comments were placeholders for possible future changes, but we don't need necessarily to change this within this PR, or at all.